jsx/Form.js

```diff
@@ -671,6 +691,7 @@ var StaticElement = React.createClass({
   mixins: [React.addons.PureRenderMixin],
   propTypes: {
     label: React.PropTypes.string,
+    columnSize: 'col-sm-9 col-sm-offset-3',
```

Comment:
:male_detective: wats this :mag_left:

Reply:
Not Sure! I will remove it my next commit.